docs/apispec-docs/src/test/scala/sttp/tapir/docs/apispec/schema/TapirSchemaToJsonSchemaTest.scala

```diff
@@ -118,4 +118,26 @@ class JsonSchemasTest extends AnyFlatSpec with Matchers with OptionValues with E
     // then
     result.asJson.deepDropNullValues shouldBe json"""{"$$schema":"http://json-schema.org/draft-04/schema#","title":"Parent","required":["innerChildField"],"type":"object","properties":{"innerChildField":{"$$ref":"#/$$defs/Child"}},"$$defs":{"Child":{"title":"MyChild","type":"object","properties":{"childName":{"type":["string","null"]}}}}}"""
   }
+
+  it should "Generate correct names for Eithers with parameterized types" in {
```

Comment:
shouldn't this test go to sth like `SchemaTest`?

Reply:
Ok, I added tests to `SchemaTest` as well. Still I think it's good keep these to make sure that the "title" field looks good now, as this was the initially reported issue.